### PR TITLE
Handle HAP detected in playbook gen and explanation

### PR DESCRIPTION
**Pull request by manstis**

### CODE REVIEW

File: packages/ansible-language-server/src/errors.ts [hunk 1]
```diff
@@ -129,6 +129,13 @@ ERRORS.addError(
     "Cloudflare rejected the request. Please contact your administrator.",
   ),
 );
+ERRORS.addError(
+  400,
+  new Error(
+    "error__wca_hap_filter_rejection",
+    "IBM watsonx Code Assistant Hate, Abuse, and Profanity filter rejected your request. Please try again.",
```
Comment (TamiTakamiya) [inline packages/ansible-language-server/src/errors.ts:136]: If the input is unchanged, re-submitting the playbook generation/explanation request will be rejected again...  I guess we need to tell the user something like "Please eliminate any doubtful phrases and try again".

Comment (manstis) [reply packages/ansible-language-server/src/errors.ts:136]: @TamiTakamiya Agreed.. I had thought that too. I'll update.

File: packages/ansible-language-server/src/errors.ts [hunk 2]
```diff
@@ -129,6 +129,13 @@ ERRORS.addError(
     "Cloudflare rejected the request. Please contact your administrator.",
   ),
 );
+ERRORS.addError(
+  400,
+  new Error(
+    "error__wca_hap_filter_rejection",
+    "IBM watsonx Code Assistant Hate, Abuse, and Profanity filter rejected your request. Please re-phrase and try again.",
```
Comment (robinbobbitt) [inline packages/ansible-language-server/src/errors.ts:136]: Given that the response from WCA doesn't spell out HAP, I'm wondering if we should be presenting this like an official entity with all caps etc. The message from WCA is:
`Our filters detected a potential problem with entities in your input. Please check your input and try again.`
What do we think about something like "Potentially harmful language was detected in your request. Please check your input and try again."

**Review by robinbobbitt — CHANGES_REQUESTED** [review]

File: packages/ansible-language-server/src/errors.ts (hunk 2 above)
Comment (manstis) [reply packages/ansible-language-server/src/errors.ts]: @robinbobbitt Updated.. Happy to go with your recommendation.

**Review by robinbobbitt — APPROVED** [review]
LGTM. Thanks @manstis !

**Pull request merged**